### PR TITLE
Standardize code documentation in api/fittings - Closes #1343

### LINKED ISSUE #1343: Apply code documentation standards in api/fittings
Parent: #1333

### PR DESCRIPTION
**Pull request by Tschakki**
### What was the problem?
Code documentation standards are not applied consistently in the project.

### How did I fix it?
Apply the same JSDoc standards in every file.

### Review checklist

* The PR solves #1343 
* Linting passes
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: api/fittings/lisk_cache.js [hunk 1]
```diff
@@ -13,9 +13,26 @@
...
+ */
+
+/**
+ * Description of the function.
+ * @func create
+ * @param {Object} fittingDef - Description of the param.
+ * @param {Object} bagpipes - Description of the param.
+ * @returns {function} lisk_cache
+ * @todo: add description of the function and its parameters
+ */
```
Comment (willclarktech) [inline api/fittings/lisk_cache.js]: In `api/controllers` we have a `@module` with `@param`s, whereas here we have two definitions. Is it important to distinguish the module from the function here?

File: api/fittings/lisk_cache.js [hunk 2]
```diff
@@ -13,9 +13,26 @@
  */
 'use strict';
 
-var modules = require('../../helpers/swagger_module_registry');
 var debug = require('debug')('swagger:lisk:cache');
+var modules = require('../../helpers/swagger_module_registry');
 
+/**
+ * Description of the module.
+ * @module
```
Comment (willclarktech) [inline api/fittings/lisk_cache.js]: Why not follow the pattern from the `api/controllers` PR with a namespace?

File: api/fittings/lisk_cache.js [hunk 3]
```diff
@@ -13,9 +13,26 @@
...
+ * @requires helpers/swagger_module_registry.getLogger
+ * @todo: add description of the module
+ */
+
+/**
+ * Description of the function.
+ * @func create
+ * @param {Object} fittingDef - Description of the param.
+ * @param {Object} bagpipes - Description of the param.
+ * @returns {function} lisk_cache
```
Comment (willclarktech) [inline api/fittings/lisk_cache.js]: Is there a way to provide a link from this to the function below in the generated documentation?

Comment (Tschakki) [reply api/fittings/lisk_cache.js]: I could not find a way to link it directly, but one could add an additional `@see`-tag, that will add a link to the function in the documentation

File: api/fittings/lisk_cache.js (hunk 1 above)
Comment (Tschakki) [reply api/fittings/lisk_cache.js]: It is not important, in fact I will restructure it a little bit according to your feedback, see next commits :)

File: api/fittings/lisk_cache.js (hunk 3 above)
Comment (Tschakki) [reply api/fittings/lisk_cache.js]: not if we are using namespaces, it seems (normally inlune links are done with `@link`)

File: api/fittings/lisk_cache.js (hunk 2 above)
Comment (Tschakki) [reply api/fittings/lisk_cache.js]: good idea :)

**Pull request merged**